src/main/java/ch/njol/skript/log/LogEntry.java

```diff
@@ -160,13 +161,17 @@ public String toFormattedString() {
 			details = OTHER_DETAILS;
 		}
 
+		String from = this.from;
+		if (!Objects.equals(from, ""))
+			from = "§7   " + from + "\n";
+
 		// Replace configured messages chat styles without user variables
 		String lineInfoMsg = replaceNewline(Utils.replaceEnglishChatStyles(lineInfo.getValue() == null ? lineInfo.key : lineInfo.getValue()));
 		String detailsMsg = replaceNewline(Utils.replaceEnglishChatStyles(details.getValue() == null ? details.key : details.getValue()));
 		String lineDetailsMsg = replaceNewline(Utils.replaceEnglishChatStyles(LINE_DETAILS.getValue() == null ? LINE_DETAILS.key : LINE_DETAILS.getValue()));
 
 		return
-			String.format(lineInfoMsg, String.valueOf(node.getLine()), c.getFileName()) +
+			String.format(lineInfoMsg, String.valueOf(node.getLine()), c.getFileName()) + // String.valueOf is needed to replace as a string
```

Comment:
```suggestion
			String.format(lineInfoMsg, String.valueOf(node.getLine()), c.getFileName()) + // String.valueOf is to convert the line number (int) to a String
```
IMO this is more clear